packages/cli/src/components/IndexPane/ClientView.tsx

```diff
@@ -0,0 +1,38 @@
+import Link from "next/link";
+
+type ClientViewProps = {
+  previews: [string, string[]][];
+};
+
+const ClientView: React.FC<ClientViewProps> = ({ previews }) => {
+  return (
+    <>
+      Client view
+      {previews.map((preview) => (
```

Comment:
This could use `entries()` to give `preview[0]` and `preview[1]` better variable names